### PR TITLE
standard: add page

### PR DESCRIPTION
**Pull request by pxgamer**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by pxgamer** [comment]
@jsonbruce, yes.

**Comment by jsonbruce** [comment]
It's great, Thanks!

And, Do you think the `--env` flag will be used usually?

**Comment by pxgamer** [comment]
I've never used it, but I guess other people may be interested in using it. Not sure why I didn't add it. :thinking:

File: pages/common/standard.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# standard
+
+> The JavaScript Standard Style tool for linting and fixing JavaScript code.
+
+- Lint all JavaScript source files in the current directory:
+
+`standard`
+
+- Lint specific JavaScript files:
```
Comment (agnivade) [inline pages/common/standard.md]: If the command allows multiple files, change the example to `file(s)`, if not, change the description to `file`.

Comment (pxgamer) [reply pages/common/standard.md]: :+1: I'll change it to `file(s)`.

Comment (jsonbruce) [reply pages/common/standard.md]: Awesome, I was touched by your rigorous attitude. :+1:  @agnivade

**Pull request merged**